### ISSUE TITLE
Integrate features from huseinzol05/Stock-Prediction-Models (#161)

### ISSUE THREAD

**Issue by sparlit** [sparlit/EQATS#1082]
Automated ingestion blueprint for huseinzol05/Stock-Prediction-Models.

- Queue #: 161
- Languages: Python, JavaScript
- Frameworks: TensorFlow, Keras, NumPy, Pandas, Scikit-learn, matplotlib, gym

## Domain classification
- Data Engines: Historical CSV ingestion, exploratory data analysis notebooks (Tesla study, outliers, overbought-oversold, which-stock), feature engineering for sentiment consensus, multivariate Monte Carlo with sentiment data, portfolio optimization notebook
- Signal & Execution Logic: Deep learning forecasting models (LSTM, GRU, Attention-is-all-you-need, CNN-Seq2seq, Seq2seq VAE variants), stacking models (autoencoder + RNN + ARIMA + XGBoost, ensemble of tree-based models), trading agents (Turtle, Moving-average, Signal rolling, Policy-gradient, Q-learning, Evolution-strategy, Double Q-learning, Recurrent Q-learning, Duel Q-learning, Actor-critic, Neuro-evolution, ABCD strategy), TensorFlow.js implementation of LSTM and signal rolling agent for browser-based inference
- Risk Engineering: Portfolio optimization (mean-variance), Monte Carlo simulation (simple, dynamic volatility, multivariate drift with sentiment), risk assessment via simulation outputs, position sizing derived from agent outputs

Blueprint file: blueprints/161-huseinzol05-Stock-Prediction-Models.md
